### ISSUE TITLE
Error when building network: Geographic extent of street layer (558527 km2) exceeds limit of 250000 km2

### ISSUE THREAD

**Issue by SRN1973** [ipeaGIT/r5r#272]
Today I wanted to build a new routing network for the whole of Germany with the currend development version of r5r. However, I encountered the above mentioned error message. Is there a workaround? In the past I was able to build the routing network without any problems. Should this indeed be a newly added constraint for whatever purposes this would according to me be big problem as it would prevent anyone form using r5r (or R5) for countrywide analysis tasks (e.g. Germany has an area of 357.588 km²). I would be glad about any suggestions for a feasible workaround (apart from the advice to use a smaller area :-) ) of this annoying (supposedly newly added) constraint.

**Comment by mvpsaraiva** [comment]
Unfortunately, this is a new constraint that was introduced recently upstream in R5 and there's nothing we can do on our side to fix it. There's no workaround for it :(
I suggest you open an issue on R5's repo on GitHub and ask them to either increase this value or allow us to change it from r5r.

**Comment by ambarja** [comment]
I have the same proble. 
Here the output of my console :point_down: 
```r
Error in r5r::setup_r5(data_path = here::here("resources/data"), verbose = FALSE) : 
  java.lang.IllegalArgumentException: Geographic extent of street layer (2840225 km2) exceeds limit of 250000 km2.
```

**Comment by ambarja** [comment]
> You could still use previous version of r5r that used an R5 version without this limit

Which version can I use?

**Comment by rafapereirabr** [comment]
I'm not entirely sure. I believe r5r v0.6.0 or v0.5.0 did not have this restriction yet

**Comment by ambarja** [comment]
> I'm not entirely sure. I believe r5r v0.6.0 or v0.5.0 did not have this restriction yet

Thank a lot @rafapereirabr , the version available for this is v0.6.0

**Comment by SRN1973** [comment]
I addressed this issue upstream in R5. Now I got a notification that the hard coded limit in R5 has been increased
from 250k sq km to 975k sq km (Addresses https://github.com/conveyal/r5/issues/815). Would it maybe be possible to timely incorporate the altered R5 base to r5r? This would allow me and possibly many others to use the current version of the great r5r library aain instead of an outdated older version with less functionality.
